packages/react-components/react-motion/library/etc/react-motion.api.md

```diff
@@ -21,7 +21,10 @@ export type AtomMotionFn<MotionParams extends Record<string, MotionParam> = {}>
 } & MotionParams) => AtomMotion | AtomMotion[];
 
 // @public
-export function createMotionComponent<MotionParams extends Record<string, MotionParam> = {}>(value: AtomMotion | AtomMotion[] | AtomMotionFn<MotionParams>): React_2.FC<MotionComponentProps & MotionParams>;
+export function createMotionComponent<MotionParams extends Record<string, MotionParam> = {}>(value: AtomMotion | AtomMotion[] | AtomMotionFn<MotionParams>): MotionComponent<MotionParams>;
```

Comment:
The diff here is just substituting the new type
```
MotionComponent<MotionParams>
```
for
```
React_2.FC<MotionComponentProps & MotionParams>
```
This follows the existing pattern with `PresenceComponent`.